onelogin_aws_cli/__init__.py

```diff
@@ -34,9 +34,10 @@ def user_choice(question, options):
 
 
 class OneloginAWS(object):
-    def __init__(self, config):
+    def __init__(self, config, args):
```

Comment:
Where are args getting passed in?

Reply:
oops! sorry.

Reply:
I fixed it in https://github.com/physera/onelogin-aws-cli/pull/17/commits/d78b4821fd4a778b89af33e6bd4f9eb1ad17f306